src/bootstrap/bin/rustc.rs

```diff
@@ -109,6 +109,8 @@ fn main() {
         // actually downloaded, so we just always pass the `--sysroot` option.
         cmd.arg("--sysroot").arg(&sysroot);
 
+        cmd.arg("-Zexternal-macro-backtrace");
```

Comment:
This snuck in here. Is there any reason to not have it on by default?

Reply:
Seems fine to me to have this always on during rustbuild as long as the UI tests don't have it. Having it on by default for regular rustc builds might be a good idea, but needs an FCP and I wasn't even able to find a tracking issue for it.